sdk-lib/src/main/java/cash/z/ecc/android/sdk/block/processor/CompactBlockProcessor.kt

```diff
@@ -646,18 +961,28 @@ class CompactBlockProcessor internal constructor(
          */
         internal const val UTXO_FETCH_RETRIES = 3
 
+        /**
+         * Latest block height fetching default attempts at retrying.
+         */
+        internal const val FETCH_LATEST_BLOCK_HEIGHT_RETRIES = 3
+
+        /**
+         * Get subtree roots default attempts at retrying.
+         */
+        internal const val GET_SUBTREE_ROOTS_RETRIES = 3
+
         /**
          * The theoretical maximum number of blocks in a reorg, due to other bottlenecks in the protocol design.
          */
         internal const val MAX_REORG_SIZE = 100
 
         /**
          * Default size of batches of blocks to request from the compact block service. Then it's also used as a default
-         * size of batches of blocks to validate and scan via librustzcash. For scanning action applies this - the
-         * smaller this number the more granular information can be provided about scan state. Unfortunately, it may
-         * also lead to a lot of overhead during scanning.
+         * size of batches of blocks to scan via librustzcash. For scanning action applies this - the smaller this
+         * number the more granular information can be provided about scan state. Unfortunately, it may also lead to
+         * a lot of overhead during scanning.
          */
-        internal const val SYNC_BATCH_SIZE = 10
+        internal const val SYNC_BATCH_SIZE = 100
```

Comment:
We should check during the RC period how this value affects performance on devices.

Reply:
Yep. We already did measurements, and it gives faster new block processing by a third, as we state in the updated changelog. The `10` was an initial value for the linear sync. But with the SbS, `100` works better. Agree on the later testing with other values.